delete_groups_response.go

```diff
@@ -0,0 +1,67 @@
+package sarama
+
+type DeleteGroupsResponse struct {
+	ThrottleTimeMs  int32
```

Comment:
For time fields we prefer to expose them as native go types, see e.g. https://github.com/Shopify/sarama/blob/0f333e6f2449e00440f7a53d3d2f006d702abe23/end_txn_response.go#L8